extensions/distributed-tracing.md

```diff
@@ -2,9 +2,9 @@
 
 This extension embeds context from
 [Distributed Tracing](https://w3c.github.io/trace-context/) so that distributed
-systems can include traces that span an event-driven system. This is the
-foundation of many other systems, such as Open Tracing, on which platforms like
-Prometheus are built.
+systems can include traces that span an event-driven system. This extensions is meant
+to contain historical data of the parent trace, in order to diagnose eventual failures of the 
+systems through tracing platforms like Prometheus.
```

Comment:
both the before and after are a bit off I think. There's little connection with prometheus here (metrics not tracing), so is is good to change it.

"This extension associates a trace context with an event in W3C format(link). When a receiver understands this format, logs can be correlated between the two. When both the sender and receiver send trace data to the same system (Jaeger, Zipkin, etc), causal diagnosis can occur as well."

Reply:
my mistake! thanks for spotting it